tgui/packages/tgui/interfaces/BotFloor.js

```diff
@@ -0,0 +1,97 @@
+import { useBackend } from '../backend';
+import { Button, LabeledList, NoticeBox, Section, Box } from '../components';
+import { ButtonCheckbox } from '../components/Button';
+import { Window } from '../layouts';
+import { BotStatus } from './common/BotStatus';
+
+export const BotFloor = (props, context) => {
+  const { act, data } = useBackend(context);
+  const {
+    noaccess,
+    painame,
+    hullplating,
+    replace,
+    eat,
+    make,
+    fixfloor,
+    nag_empty,
+    magnet,
+    tiles_amount,
+  } = data;
+  return (
+    <Window resizable>
+      <Window.Content scrollable>
+        <BotStatus />
+        <Section title="Floor Settings">
+          <LabeledList>
```

Comment:
I think some margin increase on the bottom would make this part look better, but thats just my opinion
```suggestion
          <LabeledList mb="3px">
```

Reply:
let me quickly test this. thx